<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>591</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>767</width>
    <height>591</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>767</width>
    <height>591</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>701</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: qlineargradient(spread:pad, x1:1, y1:2, x2:0, y2:0, stop:0 rgba(195,34,127), stop:0.502488 rgba(195,34,127,1), stop:1 rgba(253,187,45,1));


border-radius: 15px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>301</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;
box-shadow: 10px 5px 5px black;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Futura Maxi CG Bold&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;LOGIN&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entrar_pushbutton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#entrar_pushbutton{
	background-color:#c3227f;
	color: white;
	border-radius: 5px;
}

QPushButton#entrar_pushbutton:pressed{
	padding-left: 5px;
	padding-top:5px;
	background-color: rgba(255, 131, 145, 255);
 	background-position: calc(100% - 10px)center;
}

QPushButton#entrar_pushbutton:hover{
	background-color: #fdbb2d;
}</string>
    </property>
    <property name="text">
     <string>Entrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_lineedit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #c3227f;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass_lineedit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #c3227f;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Contrase√±a</string>
    </property>
   </widget>
   <widget class="QLabel" name="aviso_lineedit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
